packages/mask/src/extension/popups/pages/Wallet/TokenDetail/index.tsx

```diff
@@ -119,6 +120,7 @@ const TokenDetail = memo(() => {
 
     return (
         <>
+            <NormalHeader title={t('popups_assets')} />
```

Comment:
If you write it this way, everytime you switch the page, the NormalHeader will lost it's all internal state.